server/webapp/WEB-INF/applicationContext-acegi-security.xml

```diff
@@ -180,6 +180,7 @@
     <!-- End CAS Stuff -->
 
     <bean id="removeAdminPermissionFilter" class="com.thoughtworks.go.server.security.RemoveAdminPermissionFilter" autowire="autodetect"/>
+    <bean id="artifactsFilter" class="com.thoughtworks.go.server.security.ArtifactsFilter" autowire="autodetect"/>
```

Comment:
I prefer `artifactsFilter` to that. :) The one I suggested seems easier (for me) to understand what is happening. In this case, I'll need to do some mental gymnastics to try and understand what it means.

The problem, imo, is that the "ArtifactRequestingNonArtifact" is the part you want someone to read together and then the "DenialFilter" adds to that. However, when I read it first, I read it as: "ArtifactRequesting" (pause) "NonArtifactDenialFilter" - and got lost.